core/src/main/java/com/google/cloud/sql/core/CloudSqlInstance.java

```diff
@@ -163,45 +210,88 @@ String getPreferredIp(List<String> preferredTypes, long timeoutMs) {
    * new refresh is already in progress. If successful, other methods will block until refresh has
    * been completed.
    */
-  void forceRefresh() {
+  boolean forceRefresh() {
```

Comment:
Are we using this return value?

Reply:
I fixed the tests and changed this back to return void.